bots/models/run_conversation.py

```
@@ -0,0 +1,98 @@
+    )
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    objects = RunConversationQuerySet.as_manager()
+
+    class Meta:
+        indexes = [
+            models.Index(fields=["workflow", "workspace", "surface", "-updated_at"]),
+            models.Index(fields=["workflow", "uid", "surface", "-updated_at"]),
+        ]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)